addons/hr_holidays/__manifest__.py

```diff
@@ -65,7 +65,13 @@
             'hr_holidays/static/src/models/*/*.js',
         ],
         'web.assets_backend': [
-            'hr_holidays/static/src/js/time_off_calendar.js',
+            'hr_holidays/static/src/js/time_off_calendar_controller.js',
+            'hr_holidays/static/src/js/time_off_calendar_popover.js',
+            'hr_holidays/static/src/js/time_off_calendar_popover_renderer.js',
+            'hr_holidays/static/src/js/time_off_calendar_renderer.js',
+            'hr_holidays/static/src/js/time_off_calendar_views.js',
+            'hr_holidays/static/src/js/time_off_calendar_employee_controller.js',
+            'hr_holidays/static/src/js/time_off_calendar_employee_views.js',
```

Comment:
I'm afraid this (splitting into multiple files) can't be done in 14.5, it needs be done in master

Reply:
WIP :) 
The time_off_calendar_employee is for customizing the calendar when coming from the hr.employee form smart button. It requires small changes like:
- Allocation Request button is called New Allocation Request
- The employee_id, via active_ids, is added in the domain as well as in the context for creating time off and allocations sepcific to the employee

Reply:
Done, thanks :)